tools/Microsoft.Health.Extensions.BuildTimeCodeGenerator/Sql/SqlVisitor.cs

```diff
@@ -218,7 +218,21 @@ private static string GetTableTypeNameWithoutVersionSuffix(SchemaObjectName obje
 
         protected MemberDeclarationSyntax CreatePropertyForColumn(ColumnDefinition column)
         {
-            string normalizedSqlDbType = Enum.Parse<SqlDbType>(column.DataType.Name.BaseIdentifier.Value, true).ToString();
+            string normalizedSqlDbType = null;
+            var sqlDbTypeToNormalize = column.DataType.Name.BaseIdentifier.Value;
+
+            if (Enum.TryParse(sqlDbTypeToNormalize, true, out SqlDbType sqlDbType))
+            {
+                normalizedSqlDbType = sqlDbType.ToString();
+            }
+            else
+            {
+                if (string.Compare(sqlDbTypeToNormalize, "RowVersion", StringComparison.InvariantCultureIgnoreCase) == 0)
+                {
+                    // The timestamp data type is a synonym for the rowversion data type. Its syntax is now deprecated in SQL.
+                    normalizedSqlDbType = nameof(SqlDbType.Timestamp);
```

Comment:
Would it make more sense to do `SqlDbType.Timestamp.ToString()` since that's essentially what we're doing above?

Reply:
That makes sense - fixed in 96e92f1.